.github/workflows/Deploy Guessing Game v1.yml

```diff
@@ -0,0 +1,41 @@
+name: Deploy Number Guessing Game
+
+on:
+  workflow_dispatch:     
+ 
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    
+    - name: Setup Node
+      uses: actions/setup-node@v3
+      with:
+        node-version: '16'
+        
+    - name: Install dependencies
+      run: npm i
+      working-directory: ./number-guessing-game
+
+    - name: Create dummy properties
+      run: | 
+       echo "APP_DEV_PK=73315877021c8545736bc16beafd9b42307f2b7009a80cb0d1e412f1a46754d4" > .env
+       echo "END_USR_PK=6dd788b1f967133d89f5815e7aff70c97be3f13f6e6ae51fa12af7c5df0fe09e" >> .env
+       echo "ARB_API_KEY=djdjbfsdbksdjfksdbfksdbfsk" >> .env 
+      working-directory: ./number-guessing-game
```

Comment:
Whats this ?

Reply:
We had it earlier for running GG on other network's, but looks like we don't need it anymore or cant use it as we removed the feature to prevent people from running on mainnet or other networks & loose funds.